h2o-bindings/bin/gen_R.py

```diff
@@ -176,6 +170,30 @@ def gen_module(schema, algo, module):
         yield "         x <- setdiff(colnames(training_frame), event_column)"
```

Comment:
blocks above and below were moved around because of this statement that was using training_frame BEFORE it was actually validated

Reply:
This actually makes sense even as a standalone refactor.